protocol/protocol.go

```diff
@@ -853,6 +855,17 @@ func VerifyReply(versionPreference []Version, replyBytes, publicKey []byte, nonc
 		return midp, radi, errUnsupportedVersion([]Version{responseVer})
 	}
 
+	// Make sure the NONC tag is present and indicates the same nonce as sent in request
+	if versionIETF {
+		responseNonc, ok := reply[tagNONC]
+		if !ok {
+			return midp, radi, errors.New("protocol: response is missing NONC tag")
+		}
+		if !bytes.Equal(nonce, responseNonc) {
```

Comment:
nit: "Nonc" should be capitalized to match `tagNONC`. Or go with `responseNonse`?

Reply:
Fixed by 96d8bf00b783f1edc66ea04ceba5dd12bc928de2.